index.js

```diff
@@ -2,27 +2,59 @@
 
 var request = require('request');
 var async = require('async');
+var cheerio = require('cheerio');
 
 var host = 'http://www.generalfil.es/';
 var url = host + 'files-a/';
-var options = '/?filter=_Video_';
 
-
-var getFile = function(link, cb){
-  request(host + link, function(err, response, body) {
+var getFile = function (link, cb) {
+  request(host + link, function (err, response, body) {
     try {
       var finalLink = JSON.parse(body).link;
       cb(null, finalLink);
     } catch (e) {
-      cb(new Error('Error on response parsing for ' + host+link));
+      cb(new Error('Error on response parsing for ' + host + link));
     }
   });
 };
 
-var gf = function(title, cb) {
+var gf = function (title, options, cb) {
```

Comment:
Also - maybe we can check if second argument is `Object` or `String`, so user doesn't need to pass empty objects? Something like:
```javascript
if (typeof options !== 'Object' && typeof options === 'Function') {
  cb = options;
}
```

so then when user will not want to use any options, he can still call the function with just `title` and `callback`, so second argument will become `callback`. I'm not sure if I explained it clearly enoughm pls ask if you'll have any questions. 

BTW, great job! :+1: